project_4.html

```diff
@@ -0,0 +1,545 @@
+			</ol>
+
+			<h3 class="mb-3" id="pj-heading-checkpoint2">Checkpoint 2</h3>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Build the circuit from <strong>Schematic B</strong> (LED Dimmer) on your breadboard. Your goal is to make the LED grow brighter as you twist the potentiometer clockwise.
+					<div class="alert alert-info" role="alert">
+						Use the <strong>pseudo code below</strong> to program the circuit.
+					</div>
+					<div class="alert alert-info" role="alert">
+						These functions may be helpful:  <code class="pj-code language-arduino">Serial.begin()</code>, <code class="pj-code language-arduino">pinMode()</code>,
+						<code class="pj-code language-arduino">analogRead()</code>, <code class="pj-code language-arduino">analogWrite()</code>
+					</div>
+					<pre class="mt-3"><code class="pj-code language-arduino">// Assign variable to pin number for LED
+// Assign variable to pin number for Potentiometer
+
+void setup() {
+	// Configure the LED pin's behavior to OUTPUT
+	// Configure the Potentiometer's pin behavior to INPUT
+	// Configure the Serial baud rate
+}
+						
+void loop() {
+	// Read potentiometer pin value
+	// Set LED pin to the potentiometer pin value					
+}</code></pre>
+				</li>
+				<li class="list-group-item">
+					Upload your sketch to the Arduino Nano, and verify that the circuit meets the design requirements.
+				</li>
+			</ol>
+
+			<h3 class="mb-3" id="pj-heading-checkpoint3">Checkpoint 3</h3>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Build the circuit from <strong>Schematic C</strong> (RGB LED Dimmer) on your breadboard. No pseudo code this time! You are on your own.
+				</li>
+				<li class="list-group-item">
+					Upload your sketch to the Arduino Nano, and verify that the circuit meets the design requirements.
+				</li>
+			</ol>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-deliverables">Deliverables <span class="fw-light">(Enrolled Students
+					Only)</span></h2>
+			<p>
+				Students enrolled in the course must submit the following deliverables to the corresponding Canvas
+				course assignment.
+			</p>
+			<p>
+				Place the following files in a single folder:
+			</p>
+
+			<ul class="mb-4 list-group">
+				<li class="list-group-item">
+					<i class="bi bi-check2-square me-2"></i>
+					Video of the RGB LED Dimmer (Schematic C) on a breadboard
+					<div class="alert alert-info mt-2" role="alert">
+						In the video, use the potentiometers to create the colors yellow, magenta, and cyan.
+					</div>
+					<strong>Compress</strong> the folder to a <strong>zip</strong> file and rename the file using the format “<strong>ops_project#_lastname_firstname.zip</strong>” Then, submit the <strong>zip</strong> file to the Project 4 Canvas assignment.
+			</ul>
+		</main>
+	</div>
+	<!-- ======= Footer ======= -->
+	<script src="./js/ops_footer.js"></script>
+	<!-- End Footer -->
+</body>
+
+</html>
```

Comment:
We have a bug here. This <\pre\> tag needs to have a set width, so it doesn't push the other content off the right side of the viewport. Would you turn this into an issue?

For now replace the opening tag with <\pre style="white-space: pre-wrap;"\>

Do the same for the one above.